src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -375,8 +375,9 @@ struct SpecifiedStartTime : Tags::StartTime, db::SimpleTag {
   using option_tags = tmpl::list<OptionTags::StartTime>;
 
   static constexpr bool pass_metavariables = false;
-  static double create_from_options(const double start_time) noexcept {
-    return start_time;
+  static double create_from_options(
```

Comment:
This is just a change to fix a problem that I didn't notice from a previous PR because this tag wasn't constructed from options previously -- the `StartTime` options tag returns a `std::optional`, but the contents here mean that it will error if the user specifies `Auto`, because the start time is required to be specified.